### PR TITLE
Sprint 7/issue 61#bug home page

### PR DESCRIPTION
**Pull request by DevLuis123**
carousel error in index fixed

### CODE REVIEW

**Review by PudinDeveloper — APPROVED** [review]

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<div class="dark:bg-gray-800 h-auto">
+<div class="dark:bg-gray-800 h-auto ">
   <div>
     <%= render 'shared/navbar' %>
   </div>
   <div class="content">
-    <div class="carrusel mt-1">
+    <div class="carrusel">
```
Comment (MauricioNC) [inline app/views/home/index.html.erb:6]: Did you mean carousel?

File: app/assets/stylesheets/home_page.css [hunk 2]
```diff
@@ -1,31 +1,27 @@
 .carrusel {
```
Comment (MauricioNC) [inline app/assets/stylesheets/home_page.css:1]: Did you mean carousel?

File: app/views/profiles/show.html.erb [hunk 3]
```diff
@@ -49,7 +49,6 @@
     <% end %>
     <div class="action">
         <%= link_to 'Edit Profile', edit_profile_path, class:"mr-7" %>
```
Comment (MauricioNC) [inline app/views/profiles/show.html.erb:51]: Is this indentation size same as others?

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/home/index.html.erb (hunk 1 above)
Comment (DevLuis123) [reply app/views/home/index.html.erb:6]: I have fixed it, sorry!!

File: app/assets/stylesheets/home_page.css (hunk 2 above)
Comment (DevLuis123) [reply app/assets/stylesheets/home_page.css:1]: same!!

File: app/views/profiles/show.html.erb (hunk 3 above)
Comment (DevLuis123) [reply app/views/profiles/show.html.erb:51]: no, it isn't, but its fixed

File: app/assets/stylesheets/home_page.css (hunk 2 above)
Comment (MauricioNC) [reply app/assets/stylesheets/home_page.css:1]: Are you sure? I just see the same code

**Review by MauricioNC — APPROVED** [review]

**Review by STR1ND3R — APPROVED** [review]

**Pull request merged**